<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>326</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>远程桌面连接</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>331</height>
    </rect>
   </property>
   <widget class="QWidget" name="defaultWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>151</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>281</width>
       <height>64</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item row="1" column="1">
       <widget class="QLabel" name="accountTxtLable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>未指定</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="accountLable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>用户名:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="cmpLineEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="cmpLable">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>计算机(C):</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="defaultBtnWidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>7</y>
        <width>369</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="QToolButton" name="optionToolBtn">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="text">
          <string>选项</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::DownArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>134</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="connBtn">
           <property name="text">
            <string>连接</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="helpBtn">
           <property name="text">
            <string>帮助</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="defaultLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>当您连接时，向您询问凭证数据。</string>
     </property>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>defaultLabel</zorder>
    <zorder>defaultBtnWidget</zorder>
   </widget>
   <widget class="QWidget" name="optionWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>321</height>
     </rect>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>281</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>常规</string>
      </attribute>
      <widget class="QGroupBox" name="generalGroupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>371</width>
         <height>141</height>
        </rect>
       </property>
       <property name="title">
        <string>登录设置</string>
       </property>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>331</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="accountLable_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>用户名:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="cmpLable_2">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>计算机(C):</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="cmpLineEdit_2"/>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="accountTxtLable_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>未指定</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="defaultLabel_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>191</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>当您连接时，向您询问凭证数据。</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="saveRdpCheckBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>允许我保存凭据。</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="connSetGroupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>160</y>
         <width>371</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>连接设置</string>
       </property>
       <widget class="QLabel" name="connSetLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>331</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>将当前连接设置保存到RDP文件或者打开一个已保存的连接。</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>40</y>
          <width>239</width>
          <height>25</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="saveBtn">
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveasBtn">
           <property name="text">
            <string>另存为</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="openBtn">
           <property name="text">
            <string>打开</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>显示</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>361</width>
         <height>231</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="showGroupBox">
          <property name="title">
           <string>显示配置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="colorGroupBox">
          <property name="title">
           <string>颜色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="connBarCheckBox">
          <property name="text">
           <string>全屏显示时显示连接栏。</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>本地资源</string>
      </attribute>
      <widget class="QGroupBox" name="vedioGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>11</y>
         <width>359</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>语音设置</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="keyGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>359</width>
         <height>75</height>
        </rect>
       </property>
       <property name="title">
        <string>键盘</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="resGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>149</y>
         <width>359</width>
         <height>101</height>
        </rect>
       </property>
       <property name="title">
        <string>本地设备和资源</string>
       </property>
       <widget class="QLabel" name="resLabel">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>311</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>选择您要在远程会话中使用的设备和资源。</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="printCheckBox">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>打印机</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="cliCheckBox">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>60</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>剪切板</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>程序</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>体验</string>
      </attribute>
      <widget class="QGroupBox" name="perGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>351</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>性能</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="reconnCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>271</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>如果连接中断，则重新连接。</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>高级</string>
      </attribute>
      <widget class="QGroupBox" name="authGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>351</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string>服务器身份验证</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>301</width>
          <height>71</height>
         </rect>
        </property>
        <property name="lineWidth">
         <number>4</number>
        </property>
        <property name="midLineWidth">
         <number>4</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;服务器身份验证会验证是都正在连接到预期远程计算机。&lt;/p&gt;&lt;p&gt;连接所需的验证强度由系统安全策略确定。&lt;/p&gt;&lt;p&gt;如果服务器身份验证失败：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QComboBox" name="authComboBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>连接并且不显示警告</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>不连接</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>显示警告</string>
         </property>
        </item>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="optionBtnWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>7</y>
        <width>369</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <widget class="QToolButton" name="optionToolBtn_2">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="text">
          <string>选项</string>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::UpArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>134</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="connBtn_2">
           <property name="text">
            <string>连接</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="helpBtn_2">
           <property name="text">
            <string>帮助</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
